### DISCUSSION TITLE
unsupported device with Zigbee model 'TS0601' and manufacturer name ' _TZE204_myd45weu'

### DISCUSSION THREAD

**Discussion by mathgoy** [Koenkk/zigbee2mqtt#23210] [Devices]
Hi, I got this device:
https://www.aliexpress.us/item/1005005083140791.html?spm=a2g0o.order_list.order_list_main.5.599c1802b9lAtc&gatewayAdapt=4itemAdapt

which comes with this error message after integration:
```
Warning 2024-07-30 11:04:10Received message from unsupported device with Zigbee model 'TS0601' and manufacturer name '_TZE204_myd45weu'
Warning 2024-07-3011:04:10Please see: https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html
```

The device looks like any other Tuya TS0601 (or GXM-01) except it is from the brand GIEX.

It looks eactly like this (same packaging, same device)
https://smarthomescene.com/reviews/tuya-zigbee-plant-soil-sensor-gxm-01-review/

**Comment by Koenkk** [comment]
`_TZE204_myd45weu` will be supported in tomorrows release.
